goober.d.ts

```diff
@@ -5,12 +5,12 @@ export = goober;
 export as namespace goober;
 
 declare namespace goober {
-    function styled (tag: string): tagged;
+    function styled <T = any>(tag: string): tagged<T>;
```

Comment:
Ha, tried to paste your changes and see how it all works together but no luck. 🤔 
![Screenshot 2019-10-03 at 11 12 41](https://user-images.githubusercontent.com/3405161/66110163-e769b900-e5ce-11e9-932a-a849f32b3a8f.png)

Reply:
Dang I was hoping it was that easy. I have some time this morning so I'll take a closer look.

Reply:
I had placed the generics improperly. Fixed!